tests/functional/delete_account.js

```diff
@@ -101,39 +96,25 @@ define([
         .end();
     },
 
-    'visit delete account directly - no back button': function () {
```

Comment:
Since this is no longer possible, the [BackMixin](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-5793cc2a765a7665d71904a30e7b860bR110) should be removed from the view. (this comment goes for any other view that no longer has a back button)